flake.lock

```diff
@@ -2997,11 +2997,11 @@
     "flake-compat_4": {
       "flake": false,
       "locked": {
-        "lastModified": 1650374568,
-        "narHash": "sha256-Z+s0J8/r907g149rllvwhb4pKi8Wam5ij0st8PwAh+E=",
+        "lastModified": 1668681692,
+        "narHash": "sha256-Ht91NGdewz8IQLtWZ9LCeNXMSXHUss+9COoqu6JLmXU=",
```

Comment:
It's still modified. Just copy the contents from the target branch.

Reply:
sorry! should have had a better look at the other changes, thought they were ok. It's reset to the old version now